SU-SWS/decanter
# 5 Lines Lockup + Variants

# READY FOR REVIEW

# Summary
- Refactors the lockup to support all 5 lines
- Changes the concept of variants to be a single set of options instead of having variants on individual  elements
- Simplifies the template markup logic
- Added whitespace controls to twig template

# Needed By (Date)
- ?

# Urgency
- Medium: A part of our Drupal integration is reliant on this.

# Steps to Test

1. Check out this branch
2. Build the style guide
3. Review the lockup component options
4. Compare options against: https://docs.google.com/document/d/14_92mZHRocYWwFQfMWdTz5kQpT1QoSGj4AazAsRM5mc/edit
5. Compare styles against identity guide
6. Review code

# Affected Projects or Products
- Drupal integration
- Wordpress integration

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
